src/core/a-entity.js

```diff
@@ -278,7 +278,11 @@ var proto = Object.create(ANode.prototype, {
   initComponent: {
     value: function (name, data, isDependency) {
       var component;
+      var componentInfo = name.split('__');
+      var id = componentInfo[1];
```

Comment:
I think the id has to be still easily accessible. Each component is going to handle multiplicity in a different way and it's nice to have access to the id directly without having to parse any string. I can add this.name, this.attrName, this.id to the component.